kedro/runner/parallel_runner.py

```diff
@@ -256,6 +256,11 @@ def _run(  # noqa: too-many-locals,useless-suppression
 
         """
         # noqa: import-outside-toplevel,cyclic-import
+        if not self._is_async:
+            self._logger.info(
+                "Asynchronous mode is disabled for loading and saving data. Use the --async flag "
```

Comment:
Somehow this reads like "We disabled asynchronous mode for loading and saving data on your behalf" (but maybe it's just me). What about

```suggestion
                "Using synchronous mode for loading and saving data. Use the --async flag "
```

Reply:
That's a fair point. I'll update the message.